With our package's Apache configuration, we will expose node-restore under path /restore.
Therefore we are adjusting paths in the Webfinger response here.
--- a/lib/controllers/web_finger.js
+++ b/lib/controllers/web_finger.js
@@ -7,7 +7,7 @@ WebFinger.PROTOCOL_VERSION = 'draft-dejo

 WebFinger.action('hostMeta', function(pathname, extension) {
   var resource = this.params.resource,
-      origin   = this.getOrigin(),
+      origin   = this.getOrigin() + "/restore",
       json     = (extension === '.json'),
       useJRD   = (pathname === 'webfinger'),
       useJSON  = useJRD || (pathname === 'host-meta' && json),
